itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/BaseReplicationAcrossInstances.java

```diff
@@ -55,14 +56,15 @@ static void internalBeforeClassSetup(Map<String, String> overrides, Class clazz)
       throws Exception {
     conf = new HiveConf(clazz);
     conf.set("dfs.client.use.datanode.hostname", "true");
-    conf.set("hadoop.proxyuser." + Utils.getUGI().getShortUserName() + ".hosts", "*");
```

Comment:
this is for impersonation according to https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/Superusers.html, don't we want to test this scenario anymore?

Reply:
Hey @abstractdog, thanks for the review.

Take a look at my notes here:

https://issues.apache.org/jira/browse/HIVE-24484?focusedCommentId=17369708&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17369708

tldr; These unit tests are launching two HMS within the same JVM (same class-loader) and therefore they are able to modify each other's state where it stored in static variables.  This testing cannot be done any more.